### PR TITLE
chore(upstream): add pluginPackage values to...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore(upstream): add pluginPackage values to make backstage-cli happy ([RHIDP-3236](https://issues.redhat.com//browse/RHIDP-3236))

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by yashoswalyo — COMMENTED** [review]
matomo, feedback plugins :heavy_check_mark:

**Comment by yashoswalyo** [comment]
is this PR same as #1920 ?

**Comment by nickboldt** [comment]
> is this PR same as #1920 ?

same goal, different payload. #1920 has some weird changes in it and I wasn't sure if they were correct. Left comments/questions.

File: plugins/kiali-backend/package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "kiali"
+    "pluginId": "kiali",
+    "pluginPackage": "@janus-idp/backstage-plugin-kiali-backend"
```
Comment (PatAKnight) [inline plugins/kiali-backend/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-kiali-backend",
      "@janus-idp/backstage-plugin-kiali"
    ]
```

File: plugins/kiali/package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "kiali"
+    "pluginId": "kiali",
+    "pluginPackage": "@janus-idp/backstage-plugin-kiali"
```
Comment (PatAKnight) [inline plugins/kiali/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-kiali-backend",
      "@janus-idp/backstage-plugin-kiali"
    ]
```

File: plugins/kubernetes-actions/package.json [hunk 3]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.28.4",
-    "pluginId": "kubernetes"
+    "pluginId": "kubernetes",
+    "pluginPackage": "@janus-idp/backstage-scaffolder-backend-module-kubernetes"
```
Comment (PatAKnight) [inline plugins/kubernetes-actions/package.json:17]:
```suggestion
    "pluginId": "scaffolder",
    "pluginPackage": "@backstage/plugin-scaffolder-backend"
```

File: plugins/notifications-backend/package.json [hunk 4]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "notifications"
+    "pluginId": "notifications",
+    "pluginPackage": "@janus-idp/plugin-notifications-backend"
```
Comment (PatAKnight) [inline plugins/notifications-backend/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/plugin-notifications-backend",
      "@janus-idp/plugin-notifications"
    ]
```

File: plugins/notifications/package.json [hunk 5]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "notifications"
+    "pluginId": "notifications",
+    "pluginPackage": "@janus-idp/plugin-notifications"
```
Comment (PatAKnight) [inline plugins/notifications/package.json:17]:
```suggestion
    "pluginPackages": [
      "@janus-idp/plugin-notifications-backend",
      "@janus-idp/plugin-notifications"
    ]
```

File: plugins/ocm-backend/package.json [hunk 6]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "ocm"
+    "pluginId": "ocm",
+    "pluginPackage": "@janus-idp/backstage-plugin-ocm-backend"
```
Comment (PatAKnight) [inline plugins/ocm-backend/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-ocm-backend",
      "@janus-idp/backstage-plugin-ocm-common",
      "@janus-idp/backstage-plugin-ocm"
    ]
```

File: plugins/ocm-common/package.json [hunk 7]
```diff
@@ -14,7 +14,8 @@
   "backstage": {
     "role": "common-library",
     "supported-versions": "1.28.4",
-    "pluginId": "ocm"
+    "pluginId": "ocm",
+    "pluginPackage": "@janus-idp/backstage-plugin-ocm-common"
```
Comment (PatAKnight) [inline plugins/ocm-common/package.json:18]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-ocm-backend",
      "@janus-idp/backstage-plugin-ocm-common",
      "@janus-idp/backstage-plugin-ocm"
    ]
```

File: plugins/ocm/package.json [hunk 8]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "ocm"
+    "pluginId": "ocm",
+    "pluginPackage": "@janus-idp/backstage-plugin-ocm"
```
Comment (PatAKnight) [inline plugins/ocm/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-ocm-backend",
      "@janus-idp/backstage-plugin-ocm-common",
      "@janus-idp/backstage-plugin-ocm"
    ]
```

File: plugins/orchestrator-backend/package.json [hunk 9]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "orchestrator"
+    "pluginId": "orchestrator",
+    "pluginPackage": "@janus-idp/backstage-plugin-orchestrator-backend"
```
Comment (PatAKnight) [inline plugins/orchestrator-backend/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-orchestrator-backend",
      "@janus-idp/backstage-plugin-orchestrator-common",
      "@janus-idp/backstage-plugin-orchestrator-swf-editor-envelope",
      "@janus-idp/backstage-plugin-orchestrator"
    ]
```

File: plugins/orchestrator-common/package.json [hunk 10]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "common-library",
     "supported-versions": "1.28.4",
-    "pluginId": "orchestrator"
+    "pluginId": "orchestrator",
+    "pluginPackage": "@janus-idp/backstage-plugin-orchestrator-common"
```
Comment (PatAKnight) [inline plugins/orchestrator-common/package.json:17]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-orchestrator-backend",
      "@janus-idp/backstage-plugin-orchestrator-common",
      "@janus-idp/backstage-plugin-orchestrator-swf-editor-envelope",
      "@janus-idp/backstage-plugin-orchestrator"
    ]
```

File: plugins/orchestrator-swf-editor-envelope/package.json [hunk 11]
```diff
@@ -15,7 +15,8 @@
   "backstage": {
     "role": "web-library",
     "supported-versions": "1.28.4",
-    "pluginId": "orchestrator-swf-editor-envelope"
+    "pluginId": "orchestrator-swf-editor-envelope",
+    "pluginPackage": "@janus-idp/backstage-plugin-orchestrator-swf-editor-envelope"
```
Comment (PatAKnight) [inline plugins/orchestrator-swf-editor-envelope/package.json:19]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-orchestrator-backend",
      "@janus-idp/backstage-plugin-orchestrator-common",
      "@janus-idp/backstage-plugin-orchestrator-swf-editor-envelope",
      "@janus-idp/backstage-plugin-orchestrator"
    ]
```

File: plugins/orchestrator/package.json [hunk 12]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "orchestrator"
+    "pluginId": "orchestrator",
+    "pluginPackage": "@janus-idp/backstage-plugin-orchestrator"
```
Comment (PatAKnight) [inline plugins/orchestrator/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-orchestrator-backend",
      "@janus-idp/backstage-plugin-orchestrator-common",
      "@janus-idp/backstage-plugin-orchestrator-swf-editor-envelope",
      "@janus-idp/backstage-plugin-orchestrator"
    ]
```

File: plugins/quay-actions/package.json [hunk 13]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.28.4",
-    "pluginId": "quay"
+    "pluginId": "quay",
+    "pluginPackage": "@janus-idp/backstage-scaffolder-backend-module-quay"
```
Comment (PatAKnight) [inline plugins/quay-actions/package.json:17]:
```suggestion
    "pluginId": "scaffolder",
    "pluginPackage": "@backstage/plugin-scaffolder-backend"
```

File: plugins/matomo/package.json [hunk 14]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "matomo"
+    "pluginId": "matomo",
+    "pluginPackage": "@janus-idp/backstage-plugin-matomo"
```
Comment (PatAKnight) [inline plugins/matomo/package.json:17]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-matomo-backend",
      "@janus-idp/backstage-plugin-matomo"
    ]
```

File: plugins/matomo-backend/package.json [hunk 15]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "matomo"
+    "pluginId": "matomo",
+    "pluginPackage": "@janus-idp/backstage-plugin-matomo-backend"
```
Comment (PatAKnight) [inline plugins/matomo-backend/package.json:17]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-matomo-backend",
      "@janus-idp/backstage-plugin-matomo"
    ]
```

File: plugins/rbac-backend/package.json [hunk 16]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "rbac"
+    "pluginId": "rbac",
+    "pluginPackage": "@janus-idp/backstage-plugin-rbac-backend"
```
Comment (PatAKnight) [inline plugins/rbac-backend/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-rbac-backend",
      "@janus-idp/backstage-plugin-rbac-common",
      "@janus-idp/backstage-plugin-rbac-node",
      "@janus-idp/backstage-plugin-rbac"
    ]
```

File: plugins/rbac-common/package.json [hunk 17]
```diff
@@ -14,7 +14,8 @@
   "backstage": {
     "role": "common-library",
     "supported-versions": "1.28.4",
-    "pluginId": "rbac"
+    "pluginId": "rbac",
+    "pluginPackage": "@janus-idp/backstage-plugin-rbac-common"
```
Comment (PatAKnight) [inline plugins/rbac-common/package.json:18]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-rbac-backend",
      "@janus-idp/backstage-plugin-rbac-common",
      "@janus-idp/backstage-plugin-rbac-node",
      "@janus-idp/backstage-plugin-rbac"
    ]
```

File: plugins/rbac-node/package.json [hunk 18]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "node-library",
     "supported-versions": "1.28.4",
-    "pluginId": "rbac"
+    "pluginId": "rbac",
+    "pluginPackage": "@janus-idp/backstage-plugin-rbac-node"
```
Comment (PatAKnight) [inline plugins/rbac-node/package.json:17]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-rbac-backend",
      "@janus-idp/backstage-plugin-rbac-common",
      "@janus-idp/backstage-plugin-rbac-node",
      "@janus-idp/backstage-plugin-rbac"
    ]
```

File: plugins/rbac/package.json [hunk 19]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "rbac"
+    "pluginId": "rbac",
+    "pluginPackage": "@janus-idp/backstage-plugin-rbac"
```
Comment (PatAKnight) [inline plugins/rbac/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-rbac-backend",
      "@janus-idp/backstage-plugin-rbac-common",
      "@janus-idp/backstage-plugin-rbac-node",
      "@janus-idp/backstage-plugin-rbac"
    ]
```

File: plugins/regex-actions/package.json [hunk 20]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.28.4",
-    "pluginId": "regex"
+    "pluginId": "regex",
+    "pluginPackage": "@janus-idp/backstage-scaffolder-backend-module-regex"
```
Comment (PatAKnight) [inline plugins/regex-actions/package.json:17]:
```suggestion
    "pluginId": "scaffolder",
    "pluginPackage": "@backstage/plugin-scaffolder-backend"
```

File: plugins/scaffolder-annotator-action/package.json [hunk 21]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.28.4",
-    "pluginId": "annotator"
+    "pluginId": "annotator",
+    "pluginPackage": "@janus-idp/backstage-scaffolder-backend-module-annotator"
```
Comment (PatAKnight) [inline plugins/scaffolder-annotator-action/package.json:17]:
```suggestion
    "pluginId": "scaffolder",
    "pluginPackage": "@backstage/plugin-scaffolder-backend"
```

File: plugins/servicenow-actions/package.json [hunk 22]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.28.4",
-    "pluginId": "servicenow"
+    "pluginId": "servicenow",
+    "pluginPackage": "@janus-idp/backstage-scaffolder-backend-module-servicenow"
```
Comment (PatAKnight) [inline plugins/servicenow-actions/package.json:17]:
```suggestion
    "pluginId": "scaffolder",
    "pluginPackage": "@backstage/plugin-scaffolder-backend"
```

File: plugins/sonarqube-actions/package.json [hunk 23]
```diff
@@ -13,7 +13,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.28.4",
-    "pluginId": "sonarqube"
+    "pluginId": "sonarqube",
+    "pluginPackage": "@janus-idp/backstage-scaffolder-backend-module-sonarqube"
```
Comment (PatAKnight) [inline plugins/sonarqube-actions/package.json:17]:
```suggestion
    "pluginId": "scaffolder",
    "pluginPackage": "@backstage/plugin-scaffolder-backend"
```

File: plugins/topology-common/package.json [hunk 24]
```diff
@@ -14,7 +14,8 @@
   "backstage": {
     "role": "common-library",
     "supported-versions": "1.28.4",
-    "pluginId": "topology"
+    "pluginId": "topology",
+    "pluginPackage": "@janus-idp/backstage-plugin-topology-common"
```
Comment (PatAKnight) [inline plugins/topology-common/package.json:18]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-topology-common",
      "@janus-idp/backstage-plugin-topology",
    ]
```

File: plugins/topology/package.json [hunk 25]
```diff
@@ -12,7 +12,8 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "topology"
+    "pluginId": "topology",
+    "pluginPackage": "@janus-idp/backstage-plugin-topology"
```
Comment (PatAKnight) [inline plugins/topology/package.json:16]:
```suggestion
    "pluginPackages": [
      "@janus-idp/backstage-plugin-topology-common",
      "@janus-idp/backstage-plugin-topology",
    ]
```

**Review by PatAKnight — COMMENTED** [review]
I found the [doc](https://github.com/backstage/backstage/blob/master/beps/0009-plugin-metadata/README.md#package-relationships) on what the requirements are for the various plugins. It seems like we are to include all plugins that are associated with one another and module plugins are to reference the backend plugin that they are built off of. Left some review comments to try to get everything in line.

File: plugins/topology/package.json [hunk 26]
```diff
@@ -12,7 +12,11 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "topology"
+    "pluginId": "topology",
+    "pluginPackages": [
+      "@janus-idp/backstage-plugin-topology-common",
+      "@janus-idp/backstage-plugin-topology",
```
Comment (nickboldt) [inline plugins/topology/package.json:18]:
```suggestion
      "@janus-idp/backstage-plugin-topology"
```

File: plugins/topology-common/package.json [hunk 27]
```diff
@@ -14,7 +14,11 @@
   "backstage": {
     "role": "common-library",
     "supported-versions": "1.28.4",
-    "pluginId": "topology"
+    "pluginId": "topology",
+    "pluginPackages": [
+      "@janus-idp/backstage-plugin-topology-common",
+      "@janus-idp/backstage-plugin-topology",
```
Comment (nickboldt) [inline plugins/topology-common/package.json:20]:
```suggestion
      "@janus-idp/backstage-plugin-topology"
```

**Review by PatAKnight — APPROVED** [review]
/lgtm

**Pull request merged**